### DISCUSSION TITLE
Debugging MailScanner / Cron in 6.4: How to?

### DISCUSSION THREAD

**Discussion by diderich** [YetiForceCompany/YetiForceCRM#16368] [How to]
Since having updated to 6.4, I have an issue with the MailScanner in perticular, and the cron job in general.

When I manually request a mail scanner `My home page -> Software configuration -> Logs -> Mail – Download history`, I get the error message:
![cron](https://user-images.githubusercontent.com/10230144/185752607-7995eb77-f2cd-4952-9030-5cb8b38f31b8.jpg)

No e-mail gets bound and no entry is written into the table `vtiger_ossmails_log` as was the case before updating to 6.4.

There are not errors in `cache/logs/errors.log` or in `cache/logs/cron` (with `$DEBUG_CRON = true` in `Debug.php`).

When I run the cron job manually `php cron.php`, I get no error
![cron2](https://user-images.githubusercontent.com/10230144/185752748-9ace08dc-1c0b-4dd7-a895-09f35a835e7d.jpg)

No e-mail gets bound and no entry is written into the table `vtiger_ossmails_log`.

As I suspect this is an issue on my side, I was wondering how I could debug the problem to find the issue.

Any suggestions?
Thanks in advance.

**Comment by mariuszkrzaczkowski** [comment]
Create issue, attach the required information because there is not enough information
